projects/angular/src/forms/styles/_properties.forms.scss

```
@@ -96,6 +96,12 @@
       // Range
       --clr-forms-range-progress-fill-color: var(--clr-color-action-600);
       --clr-forms-range-track-color: var(--clr-color-neutral-200);
+
+      // Datalist
+
+      --clr-forms-datalist-caret-color: #{variables.$clr-color-neutral-600};
+      --clr-forms-datalist-caret-disabled-color: #{variables.$clr-color-neutral-600};
+      --clr-forms-datalist-caret-icon-size: #{variables.$clr_baselineRem_0_667};
```

extract the common `variables` to `projects/angular/src/forms/styles/_variables.forms.scss` and use them from there as all other components do.
```
      $clr-forms-datalist-caret-color: variables.$clr-color-neutral-600 !default;
      $clr-forms-datalist-caret-disabled-color: variables.$clr-color-neutral-600 !default;
      $clr-forms-datalist-caret-icon-size: variables.$clr_baselineRem_0_667 !default;
```
`projects/angular/src/forms/styles/_properties.forms.scss`
```suggestion
      --clr-forms-datalist-caret-color: #{forms-variables.$clr-forms-datalist-caret-color};
      --clr-forms-datalist-caret-disabled-color: #{forms-variables.$clr-forms-datalist-caret-disabled-color};
      --clr-forms-datalist-caret-icon-size: #{forms-variables.$clr-forms-datalist-caret-icon-size};
```

---

Irrelevant comment.